LaLauncher/src/main/java/com/github/postapczuk/lalauncher/InstalledAppsActivity.java

```diff
@@ -131,11 +148,17 @@ private void setActions() {
         onSwipeHandler();
     }
 
+    private String returnPackageName(int position) {
+        String[] packageReference = packageLookupAdapter.getItem(position).toString().split("[||]");
```

Comment:
Don't you think that using `String` to this is weak solution? If the app will have `||` in name, the launcher will crash. I suggest to use guava's `Pair<String, String>` in adapter if possible